aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -259,4 +275,11 @@ module aptos_framework::transaction_context {
         // expected to fail with the error code of `invalid_state(E_TRANSACTION_CONTEXT_NOT_AVAILABLE)`
         let _multisig = multisig_payload();
     }
+
+    #[test]
+    #[expected_failure(abort_code=196609, location = Self)]
+    fun test_call_raw_txn_hash() {
+        // expected to fail with the error code of `invalid_state(E_TRANSACTION_CONTEXT_NOT_AVAILABLE)`
+        let _multisig = multisig_payload();
```

Comment:
The function seems pretty innocent :-)
Curious why it is supposed to fail.

Reply:
The unit test runs in an environment without any user transaction context. In that case, the design decision has been to abort instead of returning some default value.